client/rest-high-level/src/main/java/org/elasticsearch/client/Request.java

```diff
@@ -462,6 +464,16 @@ static Request multiSearch(MultiSearchRequest multiSearchRequest) throws IOExcep
         return new Request("GET", "/_msearch", params.getParams(), entity);
     }
 
+    static Request rankEval(RankEvalRequest rankEvalRequest) throws IOException {
+        // TODO maybe indices should be propery of RankEvalRequest and not of the spec
```

Comment:
I opened a separate PR (#28341) to adress this, so depending on which one gets in first this might already be irrelevant.